### PR TITLE
web workflow file system page

### LINKED ISSUE #6884: Change URL in web file browser
This was discussed a bit in cp-dev

https://github.com/adafruit/circuitpython/blob/45ee13086a943c5a2b4d03883907f81f54e05c21/supervisor/shared/web_workflow/static/directory.html#L12

A better url to put there would be: 
https://learn.adafruit.com/getting-started-with-web-workflow-using-the-code-editor/device-setup#disabling-usb-mass-storage-3125964

And I think both this page and that guide should say that you can eject the drive as well.
I'll do a PR for this page and submit feedback on the page.

### PR DESCRIPTION
**Pull request by askpatrickw**
Fixes #6884

### CODE REVIEW

**Review by Neradoc — APPROVED** [review]
Thanks, that's a better link, using the new guide.
Note that, from a test build for the FunHouse, this increases builds by 64 bytes. I wonder if we could use a shortened URL redirect (from adafru.it or circuitpython.org) to reduce the binary size.

EDIT: like is already the case with `http://Adafru.It/mpy-update` in the "bad MPY format" error.

**Comment by askpatrickw** [comment]
If there is an Adafruit shortened URL for all Learn Guides, that would be very cool for lots of boards I bet...

File: supervisor/shared/web_workflow/static/directory.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
     <h1><a href="/"><img src="/favicon.ico"/></a>&nbsp;<span id="path"></span></h1>
-    <div id="usbwarning" style="display: none;">ℹ️ USB is using the storage. Only allowing reads. See <a href="https://learn.adafruit.com/circuitpython-essentials/circuitpython-storage">the CircuitPython Essentials: Storage guide</a> for details.</div>
+    <div id="usbwarning" style="display: none;">ℹ️ USB is using the storage. Only allowing reads. Try ejecting the CircuitPython drive. See <a href="https://learn.adafruit.com/getting-started-with-web-workflow-using-the-code-editor/device-setup#disabling-usb-mass-storage-3125964">Getting Started with Web Workflow</a> for details.</div>
```
Comment (askpatrickw) [inline supervisor/shared/web_workflow/static/directory.html:12]:
```suggestion
    <div id="usbwarning" style="display: none;">ℹ️ USB is using the storage. Only allowing reads. Try ejecting the CircuitPython drive. See <a href="https://learn.adafruit.com/3654/device-setup#disabling-usb-mass-storage-3125964">Getting Started with Web Workflow</a> for details.</div>
```

File: supervisor/shared/web_workflow/static/directory.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
     <h1><a href="/"><img src="/favicon.ico"/></a>&nbsp;<span id="path"></span></h1>
-    <div id="usbwarning" style="display: none;">ℹ️ USB is using the storage. Only allowing reads. See <a href="https://learn.adafruit.com/circuitpython-essentials/circuitpython-storage">the CircuitPython Essentials: Storage guide</a> for details.</div>
+    <div id="usbwarning" style="display: none;">ℹ️ USB is using the storage. Only allowing reads. Try ejecting the CircuitPython drive. See <a href="https://learn.adafruit.com/3654/device-setup#disabling-usb-mass-storage-3125964">Getting Started with Web Workflow</a> for details.</div>
```
Comment (askpatrickw) [inline supervisor/shared/web_workflow/static/directory.html:12]:
```suggestion
    <div id="usbwarning" style="display: none;">ℹ️ USB is using the storage. Only allowing reads. Try ejecting the CIRCUITPY drive. See <a href="https://learn.adafruit.com/3654/device-setup#disabling-usb-mass-storage-3125964">Getting Started with Web Workflow</a> for details.</div>
```

**Review by Neradoc — APPROVED** [review]

**Pull request merged**